bin/rose-date

```diff
@@ -17,162 +17,194 @@
+    ROSE_TASK_CYCLE_TIME/ISODATETIMEREF
+        Specify the current cycle time of a task in a suite. If the
+        `--use-task-cycle-time` option is set, the value of this environment
+        variable is used by the command as the reference time instead of the
+        current time.
+
+OFFSET FORMAT
+    `OFFSET` must follow the ISO 8601 duration representations such as
+    `PnW` or `PnYnMnDTnHnMnS - P` followed by a series of `nU` where `U` is
+    the unit (`Y`, `M`, `D`, `H`, `M`, `S`) and `n` is a positive integer,
+    where `T` delimits the date series from the time series if any time units
+    are used. `n` may also have a decimal (e.g. `PT5.5M`) part for a unit
+    provided no smaller units are supplied. It is not necessary to
+    specify zero values for units. If `OFFSET` is negative, prefix a `-`.
+    For example:
+
+    * `P6D` - 6 day offset
+    * `PT6H` - 6 hour offset
+    * `PT1M` - 1 minute offset
+    * `-PT1M` - (negative) 1 minute offset
+    * `P3M` - 3 month offset
+    * `P2W` - 2 week offset (note no other units may be combined with weeks)
+    * `P2DT5.5H` - 2 day, 5.5 hour offset
+    * `-P2YT4S` - (negative) 2 year, 4 second offset
+
+PARSE FORMAT
+    The format for parsing a date time point should be compatible with the
+    POSIX strptime template format (see the strptime command help), with the
+    following subset supported across all date/time ranges:
+
+    `%F`, `%H`, `%M`, `%S`, `%Y`, `%d`, `%j`, `%m`, `%s`, `%z`
+
+    If not specified, the system will attempt to parse `DATE-TIME` using
+    the following formats:
+
+    * ctime: `%a %b %d %H:%M:%S %Y`
+    * Unix date: `%a %b %d %H:%M:%S %Z %Y`
+    * Basic ISO8601: `%Y-%m-%dT%H:%M:%S`, `%Y%m%dT%H%M%S`
+    * Cylc: `%Y%m%d%H`
+
+    If none of these match, the date time point will be parsed according to
+    the full ISO 8601 date/time standard.
+
+PRINT FORMAT
+    For printing a date time point, the print format will default to the same
+    format as the parse format. Also supports the isodatetime library dump
+    syntax for these operations which follows ISO 8601 example syntax - for
+    example:
+
+    * `CCYY-MM-DDThh:mm:ss` -> `1955-11-05T09:28:00`,
+    * `CCYY` -> `1955`,
+    * `CCYY-DDD` -> `1955-309`,
+    * `CCYY-Www-D` -> `1955-W44-6`.
+
+    Usage of this ISO 8601-like syntax should be as ISO 8601-compliant
+    as possible.
+
+    Note that specifying an explicit timezone in this format (e.g.
+    `CCYY-MM-DDThh:mm:ss+0100` or `CCYYDDDThhmmZ` will automatically
+    adapt the date/time to that timezone i.e. apply the correct
+    hour/minute UTC offset.
+
+    For printing a duration, the following can be used in format
+    statements:
+
+    * `y`: years
+    * `m`: months
+    * `d`: days
+    * `h`: hours
+    * `M`: minutes
+    * `s`: seconds
+
+    For example, for a duration `P57DT12H` - `y,m,d,h` -> `0,0,57,12`
+"""
+
+import sys
+import os
+
+from isodatetime.main import main
+
+
+def main():
```

Comment:
Well that fixed the documentation tests, unfortunately it has broken the regular tests!

Sorry, this is my bad! This line redefines `main` from the import above which breaks the command.

Something like this aught to fix it:

```python
from isodatetime.main import main as iso_main
# ...
sys.exit(iso_main())
```